docs/src/pages/[platform]/connected-components/liveness/full-api.react.mdx

```diff
@@ -0,0 +1,74 @@
+import { Tabs, TabItem, Table, TableHead, TableRow, TableCell, TableBody, View } from '@aws-amplify/ui-react';
+import { ResponsiveTable, ResponsiveTableCell } from '@/components/ResponsiveTable';
+
+## Full API Reference
+
+Below is the full list of props that can be used with the `FaceLivenessDetector` component. You can also reference the [source code here](https://github.com/aws-amplify/amplify-ui/blob/main/packages/react-liveness/src/components/FaceLivenessDetector/service/types/liveness.ts#L4).
+
+<ResponsiveTable highlightOnHover>
```

Comment:
Would it be better if we don't hard code it? Import the data instead?

Reply:
+1 
Perhaps a json structure that can be iterated over so that new additions are no-op from updates here.

Reply:
I looked into iterating the types and didn't really see to do this here that would be a no-op solution. Importing the types still requires building an object with the type and iterating over the keys. But we would still need to manually add a description and the proper human readable typing. What do you all think?

I.e

```
interface ExampleInterface {
  name: string;
  age: number;
  email: string;
}

const example: ExampleInterface = {
  name: "John Doe",
  age: 30,
  email: "john.doe@example.com",
};

for (const key of Object.keys(example)) {
  // Create the react component with description and type here based on the key
}

```

Reply:
Do you think we can use something like [TypeDoc](https://github.com/TypeStrong/typedoc) to extract structured information as JSON and then consume? Then the API definition would be the single source of truth and rest we handle via build steps.

https://atanasstoyanov.medium.com/extracting-structured-type-information-2e7480ca9f83

This could be a bigger work scope and hence will not block the change for the same. But we should explore it as it would be helpful for all the components.

Reply:
yeah i think that makes sense! can look into it as a followup but as you said probably bigger than the scope of this pr